src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -657,6 +704,8 @@ public JdbcSinkConfig(Map<?, ?> props) {
         DateTimezone.valueOf(getString(DATE_TIMEZONE_CONFIG).toUpperCase());
     dateTimeZone = dateTimezoneConfig.equals(DateTimezone.UTC)
         ? TimeZone.getTimeZone(ZoneOffset.UTC) : timeZone;
+    timestampPrecisionMode =
+        TimestampPrecisionMode.valueOf(getString(TIMESTAMP_PRECISION_MODE_CONFIG).toUpperCase());
```

Comment:
we are setting the values here but we are not using them anywhere. 
could you please change the Dialect implementations to directly use these values ?